### ISSUE TITLE
Accessibility > Colorpicker should behave like a modal

### ISSUE THREAD

**Issue by Blackbaud-SteveBrush** [blackbaud/skyux-colorpicker#36]
See: https://www.w3.org/TR/wai-aria-practices/#dialog_modal

This will need a "focus trap" directive.
Related: https://github.com/blackbaud/skyux-core/issues/152
